staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -274,15 +274,22 @@ func (w *watchCache) processEvent(event watch.Event, resourceVersion uint64, upd
 
 // Assumes that lock is already held for write.
 func (w *watchCache) updateCache(event *watchCacheEvent) {
+	var toRemove bool
```

Comment:
Prefer rename as isRemove and initialize as false.

Reply:
Golang will guarantee it's initialized to false, which is the zero value for a bool type.

Reply:
I agree golang would do that but I personally do not rely on compiler since it does not save any performance.

Reply:
It's not about compiler implementation. It's defined by the go language specification.  Initializing to false here actually is a redundancy, which I don't like :)

Reply:
I found it is clear :)

Reply:
:) I will resolve this since it's personal preference.